test/jsUtils.test.ts

```diff
@@ -97,13 +97,13 @@ describe('jsUtils', () => {
   });
 
   describe('isEscKeyPress', () => {
-    it('returns true if the key number is 27', () => {
-      const result = jsUtils.isEscKeyPress(27);
+    it('returns true if the key is "Escape"', () => {
```

Comment:
same here - will this work for non-english languages?

Reply:
Tried in a Portuguese keyboard and it worked fine, not sure if others work though.

Reply:
Thats a good question, from my understanding that is a JS API that only accepts keys named in English and not caring about the locale, see: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#examples

Reply:
in that case, im ok. resolving this comment